Documentation/contributing.rst

```diff
@@ -1482,27 +1482,71 @@ GitHub issues using the process below:
 Triage process
 ^^^^^^^^^^^^^^
 
-#. Investigate the failure
-#. Search `GitHub issues <https://github.com/cilium/cilium/issues?utf8=%E2%9C%93&q=is%3Aissue+>`_
-   to see if bug is already filed. Make sure to also include closed issues in
-   your search as a CI issue can be considered solved and then re-appears.
-#. If no existing GitHub issues was found, file a new GitHub issue:
+#. Discover Jenkins failures via the jenkins-failures.sh script. It defaults to
+   the previous 24 hours but this can be modified by setting the SINCE
+   environment variable (it is a unix timestamp). The script checks the various
+   test pipelines that need triage.
+::
+
+    $ contrib/scripts/jenkins-failures.sh
+
+.. note::
+
+   You can quickly assign SINCE with statements like ``SINCE=`date -d -3days```
+
+#. Investigate the failure you are interested in and determine if it is a
+   CI-Bug, Flake, or a Regression.
```

Comment:
Make sure to operationally define what these mean; notate what constitutes a CI Bug, a flake, or a regression.